inception/inception-log/src/main/java/de/tudarmstadt/ukp/inception/log/config/EventLoggingPropertiesImpl.java

```diff
@@ -32,14 +33,12 @@ public class EventLoggingPropertiesImpl
 {
     private boolean enabled;
 
-    private Set<String> excludeEvents = Set.of( //
-            // Do not log this by default - hardly any information value
-            AfterCasWrittenEvent.class.getSimpleName(), //
-            AvailabilityChangeEvent.class.getSimpleName(), //
-            "RecommenderTaskNotificationEvent", //
-            BeforeDocumentOpenedEvent.class.getSimpleName(), //
-            PreparingToOpenDocumentEvent.class.getSimpleName(), //
-            "BrokerAvailabilityEvent", //
+    private Set<String> includePatterns = Set.of(".*"); // Default include everything
```

Comment:
I think and empty set should imply `.*`- so it should probably not be necessary to have to specify `.*` explicitly.

Reply:
Set.of() can't be empty

Reply:
There should be a `Collections.emptySet()` or simply `new HashSet<>()`.